src/main/java/org/mitre/synthea/helpers/Utilities.java

```diff
@@ -58,6 +59,12 @@ public static int getYear(long time) {
     return calendar.get(Calendar.YEAR);
   }
 
+  public static int getMonth(long time) {
+    Calendar calendar = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+    calendar.setTimeInMillis(time);
+    return calendar.get(Calendar.MONTH);
```

Comment:
This line of code returns month on a zero-based index... January is 0, and December is 11.

Reply:
Please add a unit test that checks for month equality (e.g. `==`). I suspect that you'll need to add 1 to the return value, since in the GMF people will probably enter `12` when they mean December.